lec17-rmarkdown.Rmd

```diff
@@ -1,4 +1,388 @@
+#    - PIs not willing adopt new practices and say they do not have time to relearn
+#    - Many students doing this might in a lab where the PI is not computational and does not fully
+#    appreciate the benefits. It is also hard to take the first steps as a student not knowing what
+#    the state of the art is.
+#    - Few, if any, courses in this at the university, especially outside of computer science.
+```
+
+### Further Reading
+
+- This book called [Practice Reproducible Research](https://www.practicereproducibleresearch.org/)
+has many high level examples of how reproducible research looks in practice and further reading on the topic
+- Nature has a series of articles, editorials and research on the [challenges in  reproducible research](http://www.nature.com/news/reproducibility-1.17552)
+
+## Metadata
+
+### Why is Metadata Important?
+
+From the [Mozilla Science WOW Data Reuse Template](https://github.com/mozillascience/working-open-workshop/blob/gh-pages/handouts/data_reuse_plan_template.md):
+
+> Standard Metadata: Increasingly, scientific fields are moving towards standard
+> metadata formats (data.json, data.xml, etc) to pull all the information in the
+> Data Reuse plan together in a machine readable format. Machine readable metadata
+> enables cataloging of datasets on sites like Data.gov and allows other to ask
+> questions and access your datasets using code. For example, open US govenment
+> data online is required to expose a data.json in the landing page html to be
+> listed on Data.gov, thereby facilitating data discovery. Because not all
+> researchers who are mandated to actually include data.json files, Data.gov is
+> incomplete, and simple questions like "what is the total volume of data
+> generated by US federally funded scientists?" are unanswerable.
+
+Let's go over a few of these questions:
+
+1. What do you think metadata is?
+2. When you were choosing your dataset, did you encounter problems understanding the data?
+3. If so, what would have helped you to understand?
+
+
+### What Are Best Practices in Generating Metadata?
+
+Take some time to read over the first page of the [Center for Government
+Excellence's](http://govex.jhu.edu/) ["Open Data Metadata Guide"](https://www.gitbook.com/book/centerforgov/open-data-metadata-guide/details)
+to get a better idea.
+
+Now let's browse through the sections for more specific best practices.
+
+To review, metadata includes information about how the data was collected,
+succinct descriptions of the data as well as information about when it was last
+updated are very important to let people know if this is the right dataset for them.
+Once people decide to use your dataset, things like licensing and column
+metadata become very important. Column metadata are sometimes called data dictionaries.
+Here is an example: http://www.utexas.edu/cola/redcap/_files/data_dictionary_example.jpg
```

Comment:
https://liberalarts.utexas.edu/redcap/_files/data_dictionary_example.jpg

Reply:
Also, I think if you place this in `<>`, then GitHub won't try to render the underscores as italics.

Reply:
Oh well, seems like that didn't help the italics.